indico/modules/events/surveys/templates/survey_announcement.html

```diff
@@ -0,0 +1,28 @@
+{% set survey_count = surveys|length %}
+
+<div class="infoline surveys">
+    <span class="icon icon-clipboard"></span>
+    <div class="text">
+    <div class="label">
+        {% trans %}
```

Comment:
no need to put those on their own line (and if you do, use `{% trans -%}...{%- endtrans %}`) to avoid surrounding whitespace in the extracted strings